registry/txt.go

```diff
@@ -146,6 +146,11 @@ func (im *TXTRegistry) Records(ctx context.Context) ([]*endpoint.Endpoint, error
 			continue
 		}
 		// We simply assume that TXT records for the registry will always have only one target.
+		// If there are no targets (e.g for routing policy based records in google), direct targets will be empty
+		if len(record.Targets) == 0 {
+			log.Errorf("TXT record has no targets", record.DNSName)
```

Comment:
I'm seeing this comment 

```
printf: github.com/sirupsen/logrus.Errorf call has arguments but no formatting directives (govet)
```

Not sure if it's an error, most likely a DEBUG or WARNING

Reply:
I will fix it, need to have `%s` at the end.